src/components/Tab.js

```diff
@@ -0,0 +1,70 @@
+export function createTabs(onTabChange = null) {
+  const tabContainer = document.getElementById("tab-container");
+  if (!tabContainer) {
+    console.error("탭 컨테이너가 없습니다.");
+    return;
+  }
+
+  let selectedTab = sessionStorage.getItem("selectedTab") || "popular";
+
+  function renderTabs() {
+    tabContainer.innerHTML = "";
+    const ul = document.createElement("ul");
+    ul.classList.add("tab");
+
+    const tabItems = [
+      { category: "now_playing", label: "상영 중" },
+      { category: "popular", label: "인기순" },
+      { category: "top_rated", label: "평점순" },
+      { category: "upcoming", label: "상영 예정" },
+    ];
+
+    tabItems.forEach(({ category, label }) => {
+      const li = document.createElement("li");
+      li.setAttribute("data-category", category);
+      const a = document.createElement("a");
+      a.href = "#";
+      const div = document.createElement("div");
+      div.classList.add("tab-item");
+      div.innerHTML = `<h3>${label}</h3>`;
+      a.appendChild(div);
+      li.appendChild(a);
+      ul.appendChild(li);
```

Comment:
아래처럼 구현하는 방법도 있었을텐데 `createElement`를 선택하신 이유가 궁금합니다 !

```suggestion
   <ul>
     <li>
       <a ... />
     </li>
   </ul>
```

Reply:
GPT가 이 방법이 DOM API를 직접 사용해서 XSS공격에 더 안전하다고 해서 이렇게하고,
스켈레톤 부분은 동적으로 변하는 값이 아예 없어서 그냥 가독성이 좋다고 생각한 
템플릿 리터럴 방식을 사용했습니다.
다시 코드를 보니 tabItem을 선언하고 그 값을 그대로 쓰고 있어서 템플릿 리터럴로 바꾸는게 적합할것같습니다, 감사합니다.

그리고 궁금한 부분이 있는데 코치님은 혹시 템플릿 리터럴이랑 createElement를 어떤 기준에 따라서 사용하시는지 여쭤보고 싶습니다,,!

Reply:
저는 가독성의 차이가 너무 크기 때문에 대부분 상황에서 템플릿 리터럴을 활용하는거 같아요. 말씀해주신 XSS 공격 등 취약점이 존재하지만 그걸 인지하고 신경 써서 대비한다면 충분히 방지할 수 있다고 생각합니다

Reply:
두개를 비교해서 살펴보니 정말 가독성 차이가 많이나네요,
다른 사람이 코드를 읽는 순간까지는 잘 고려하지 않았던것 같습니다,
감사합니다,,!